pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -529,7 +529,7 @@ func (e *TokensController) generateTokenIfNeeded(serviceAccount *v1.ServiceAccou
 	if liveSecret.ResourceVersion != cachedSecret.ResourceVersion {
 		// our view of the secret is not up to date
 		// we'll get notified of an update event later and get to try again
-		klog.V(2).Infof("secret %s/%s is not up to date, skipping token population", liveSecret.Namespace, liveSecret.Name)
+		klog.V(2).InfoS("Secret is not up to date, skipping token population", klog.KObj(liveSecret))
```

Comment:
Wrong format?

Reply:
Can you point out the non-compliance with the document? thanks!

Reply:
```suggestion
		klog.V(2).InfoS("Secret is not up to date, skipping token population", "secret", klog.KObj(liveSecret))
```